pageserver/src/restore_local_repo.rs

```diff
@@ -483,9 +483,9 @@ pub fn save_decoded_record(
                 },
                 rpageno,
                 lsn,
-                ZERO_PAGE,
+                Bytes::from_static(ZERO_PAGE),
             )?;
-        } else if info == pg_constants::XLOG_MULTIXACT_ZERO_OFF_PAGE {
+        } else if info == pg_constants::XLOG_MULTIXACT_ZERO_MEM_PAGE {
```

Comment:
Now that's fixing an actual bug, not just a clippy warning :-). https://github.com/zenithdb/zenith/issues/453

Let's do that as a separate PR.

Reply:
Done. See #499